py/erlpack/_unpacker.pyx

```diff
@@ -226,7 +226,13 @@ cdef class ErlangTermDecoder(object):
         """BINARY_EXT"""
         length, = struct.unpack('>L', bytes[offset:offset + 4])
         offset += 4
-        return bytes[offset:offset + length], offset + length
+        rv = bytes[offset:offset + length]
+        if self.encoding:
+            try:
+                rv = rv.decode(self.encoding)
+            except UnicodeError:
+                pass
```

Comment:
There’s merit in throwing here rather than silently failing.

Reply:
sure! wdyt about throwing in the `STRING_EXT` handler as well? It's not clear to me why that would silently continue while this would throw if the decoder is set to an encoding